src/main/java/com/blackducksoftware/integration/jira/task/BlackDuckMonitor.java

```diff
@@ -125,7 +125,7 @@ public void reschedule(final long intervalIgnored) {
         }
 
         final HashMap<String, Serializable> blackDuckJobRunnerProperties = new HashMap<>();
-        blackDuckJobRunnerProperties.put(KEY_CONFIGURED_INTERVAL_MINUTES, actualInterval);
+        blackDuckJobRunnerProperties.put(KEY_CONFIGURED_INTERVAL_MINUTES, actualInterval / 1000);
```

Comment:
I would personally prefer this conversion done in a named variable to know what numbers we're dealing with

Reply:
Oh wow, everything about the interval is very wrong.